docs/content/sips/005-manifest-redesign.md

```diff
@@ -61,31 +85,41 @@ In order to decouple triggers and components, we move triggers to new top-level
 ```toml
 [[trigger.http]]
 route = "/world"
-component = "world"
-
-[[component]]
-id = "world"
+handler = "world"
```

Comment:
Should this be `handler_component`? Handler makes sense, because that's what it does :-), but below the proposal still refers to it as a component `[component.world]`.

Reply:
Sure, it could be. Just a tradeoff between being concise and being explicit I guess :shrug:

Reply:
I think it makes it easier to talk about if it's explicit.

Reply:
Opinion alert: I don't think there is a danger of confusion with just `handler`.

Reply:
+1 to @radu-matei ....want to avoid `component` studder everywhere.

Reply:
Can you elaborate on that Brian? Are we trying to get rid of component as a Spin concept, and/or align with webassembly concepts and/or wasi-http concepts?

Reply:
I think what motivates my lean towards the concise here is that conceptually a trigger, as modeled in WIT (or generally as a component), might `import` an instance of whatever `http` component (`hello` in this example) and that `import` name expressed in WIT might look like `import handler: ...` or `import fermyon:spin/handler`. You probably wouldn't say in WIT `import handler_component: ...`.

I definitely tend to prefer explicitness but in this case (albeit without great justification) I lean towards the concise I suppose. Seeing `*_component` throughout a larger example manifest would be hard on the eyes, me thinks.

This is not a hill I'm willing to die on though. 

Alternative suggestion, what if we named the field simply `component` which would ameliorate the concise vs explicitness tension. 

🤷‍♂️

Reply:
Thanks for the input. Rereading the combined example, I can see how the 'handler' as part of a 'trigger' section, as a reference to 'who is handling this trigger' (which is a reference to a component; either inline source, or a 'component' section.) is easy enough to explain. Thx!